src/h2olog/h2olog.h

```diff
@@ -29,8 +29,24 @@
 #include <string>
 extern "C" {
 #include <time.h>
+#include <sys/socket.h>
+#include <netinet/in.h>
 }
 
+// Corresponding to the types in the BPF program.
+// In particualr, h2olog_sockaddr_storage is limited only to
+// sockaddr_in and sockaddr_in6 because `struct sockaddr_storage`
+// is too large to be put on the BPF stack (limited to 512 bytes),
+// while `sizeof(struct sockaddr_storage)` is 128 bytes in Linux/x64.
+typedef sockaddr h2olog_sockaddr;
+typedef sockaddr_in h2olog_sockaddr_in;
+typedef sockaddr_in6 h2olog_sockaddr_in6;
```

Comment:
I do not think we need these lines, do we? This header file is for C++, and IIRC there's no need to have typedefs.

Reply:
b557aaad8d0789e97f7ed05ce885c57bf7309d11

Indeed, It's no need to define the confusing type names. Now it becomes clear and tidy.